bin/cdxgen.js

```diff
@@ -121,7 +121,9 @@ const args = yargs(hideBin(process.argv))
   })
   .scriptName("cdxgen")
   .version()
-  .help("h").argv;
+  .alias("v", "version")
```

Comment:
Nice one. Could you also update the readme? I generally run with `-h` and copy-paste the full options to the readme.

Reply:
oh sure.

Reply:
:done: